Mathlib/SetTheory/Nimber/Field.lean

```diff
@@ -218,4 +217,127 @@ instance : CommRing Nimber where
 instance : IsDomain Nimber where
 instance : CancelMonoidWithZero Nimber where
 
+/-! ### Nimber division -/
+
+mutual
+
+/-- The nimber inverse `a⁻¹` is recursively defined as the smallest nimber not in the set `s`, which
```

Comment:
```suggestion
/-- The nimber inverse `a⁻¹` is mutually recursively defined as the smallest nimber not in the set `s`, which
```
or some alternate wording - ultimately I think it'd be good to mention that this is a mutually recursive definition, not just regular recursive